app/services/zendesk_sms_service.rb

```diff
@@ -10,18 +10,18 @@ def instance
   end
 
   def handle_inbound_sms(phone_number:, sms_ticket_id:, message_body:)
-    users = User.where(phone_number: phone_number)
+    intakes = Intake.where(phone_number: phone_number)
```

Comment:
this change should be coupled with the change to `create_intake_ticket_requester`